docs/docs/typography-js.md

```diff
@@ -10,7 +10,11 @@ Typography.js is a JavaScript library that allows you to explore the typographic
 
 Gatsby has the plugin `gatsby-plugin-typography` to integrate Typography.js into your project.
 
-You can install the plugin and its peer dependencies into your project by running the command `npm install gatsby-plugin-typography react-typography typography --save`
+You can install the plugin and its peer dependencies into your project by running the following command:
+
+```shell
+npm install gatsby-plugin-typography react-typography typography --save
```

Comment:
```suggestion
npm install gatsby-plugin-typography react-typography typography
```

Reply:
`--save` is no longer necessary for modern versions of npm, but overall this looks like a solid change!

Reply:
Perfect, I'm also in favor of removing `--save` but wasn't sure what the convention in Gatsby docs was.

Reply:
Some of them are still hanging around it seems :)

Reply:
i open a Issue for this:
- #22444 `docs: Suggestion: unify npm / remove --save from npm install`